packages/viewer/src/commands/streamCommands.ts

```diff
@@ -50,6 +50,20 @@ export function startStream(
   };
 }
 
+export function reconnect(
+  streamId: UUID.UUID,
+  dimensions: Dimensions.Dimensions
+): Command<Promise<Disposable>> {
+  return ({ stream }: CommandContext) => {
+    return stream.reconnect({
+      streamId: new vertexvis.protobuf.core.Uuid({
+        hex: streamId,
+      }),
```

Comment:
Wrapping this in a `new vertexvis.protobuf.core.Uuid` is unnecessary I think. You should be able to do:
```ts
{
  streamId: { hex: streamId }
}
```